.changeset/giant-buses-talk.md

```diff
@@ -0,0 +1,5 @@
+---
+'@roadiehq/catalog-backend-module-okta': patch
+---
+
+Update new package imports from `catalog-common`.
```

Comment:
The import are from `plugin-catalog-node` and they were coming from `plugin-catalog-backend`. 🤔 

```suggestion
Uses imports from the `@backstage/plugin-catalog-node` package
```